<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sc2AiAppDialog</class>
 <widget class="QMainWindow" name="Sc2AiAppDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SC2AI Manager</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="TestTab">
    <attribute name="title">
     <string>Test Local Bot</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>108</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Region</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="RegionCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Map</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="MapCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Local Bot</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="LocalBotCombo"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Remote Bot</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="RemoteBotCombo"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="LaunchButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Launch</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="HumanTab">
    <attribute name="title">
     <string>Play Vs Bot</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Region</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="HRegionCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Map</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="HMapCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Remote Bot</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="HRemoteBotCombo"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="HRaceCombo">
        <item>
         <property name="text">
          <string>Terran</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zerg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Protoss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Random</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Race</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="HLaunchButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Launch</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="UploadTab">
    <attribute name="title">
     <string>Upload Bot</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Bot Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="BotNameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Bot Excutable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="DirectoryEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Race</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="RaceCombo">
        <item>
         <property name="text">
          <string>Terran</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zerg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Protoss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Random</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>API</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="APICombo">
        <item>
         <property name="text">
          <string>Binary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Python</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Java</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DotNet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NodeJS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mono</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wine</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Additional Arguments</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="ArgsEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Debug</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="DebugCheckBox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="TestButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowseButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ConfigButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Config</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UploadButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
